bmcstored_dump_entry.hpp

```diff
@@ -159,6 +128,9 @@ class Entry : public phosphor::dump::Entry, public FileIfaces
     /** @brief Indicates whether offload in progress
      */
     bool offloadInProgress;
+
+    /** @brief map of SDEventPlus child pointer added to event loop */
+    std::unique_ptr<Child> childPtr;
```

Comment:
this is not a map

Reply:
@devenrao looks like PR is not updated with latest gerrit version?
std::map<pid_t, std::unique_ptr<Child>> childPtrMap;

Reply:
this is jut entry so why do we need map, it is per dump right

Reply:
you are correct , offload allows one at a time.